test/envs/regional-dr.yaml

```diff
@@ -27,6 +27,7 @@ templates:
     disk_size: "50g"
     feature_gates:
       - StatefulSetAutoDeletePVC=true
+      - CSIVolumeGroupSnapshot=true
```

Comment:
This need k8s 1.32, so you need to upgrade minikube to latest release 1.35.

Reply:
It's a flag in the sidecar and the controller any specific reason to havea  dependency on the kubernetes version here?

Reply:
This is not a k8s feature gate? How do we enable it then?

Reply:
it need to be enabled on the snapshot controller and volumegroupsnapshot CRD's need to be installed

Reply:
I see this is already enabled in rook 1.16:
https://github.com/ceph/ceph-csi/blob/devel/deploy/cephfs/kubernetes/csi-cephfsplugin-provisioner.yaml

So maybe we just need to update external-snapshotter to 8.2.0, and rook to 1.16, and no other change is needed?

Reply:
The featuregate still required to be enabled on the snapshot controller if are planning to use volumegroupsnapshot feature (not sure if its already enabled somewhere in this repo)

Reply:
@Madhu-1 We will add another commit patching the snapshot controller.

Reply:
sounds good 👍🏻